Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.cs

```diff
@@ -0,0 +1,170 @@
+﻿using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents the control that redistributes space between columns or rows of a Grid control.
+    /// </summary>
+    [TemplatePart(Name = SplitterName, Type = typeof(Thumb))]
+    [TemplatePart(Name = IconDisplayName, Type = typeof(Thumb))]
```

Comment:
Are these parts really needed?
Looks like one is just used for content (why not have a content property, or let this be a content control, and the other for manipulation, but why not just have manipulation on this instance  instead of on a part?